### PR TITLE
Render query modifications

### PR DESCRIPTION
**Pull request by sagarrakshe**
Updated the query builder API.

### CODE REVIEW

File: bigquery/client.py [hunk 1]
```diff
@@ -396,6 +396,105 @@ def get_table(self, dataset, table):
...
+        except HttpError as e:
+            logging.error(('Cannot update table {0}.{1}\n'
+                           'Http Error: {2}').format(dataset, table,
+                                                     e.content))
+            if self.swallow_results:
+                return False
+            else:
+                return {}
+
+    def patch_table(self, dataset, table, schema, expiration_time=None):
```
Comment (tylertreat) [inline bigquery/client.py]: `patch_table` already exists: https://github.com/tylertreat/BigQuery-Python/blob/master/bigquery/client.py#L487

File: bigquery/client.py [hunk 2]
```diff
@@ -396,6 +396,105 @@ def get_table(self, dataset, table):
 
         return table
 
+
+    def update_table(self, dataset, table, schema, expiration_time=None):
```
Comment (tylertreat) [inline bigquery/client.py]: `update_table` already exists: https://github.com/tylertreat/BigQuery-Python/blob/master/bigquery/client.py#L445

**Comment by tylertreat** [comment]
It looks like the client.py changes don't belong with the `query_builder.py` changes.

**Comment by sagarrakshe** [comment]
Removed the *client.py *changes from the *query_builder.py. *

Thanks.
Sagar

**Pull request merged**